client/app/login/login.html

```diff
@@ -0,0 +1,44 @@
+<div class="container">
+	<div class="row">
```

Comment:
we have inconsistent spacing here. there's a mix of one tab and two tabs (or 4 and 8 spaces, can't tell). It's useful for clean code and debugging in the future